spec/features/media_viewer_spec.rb

```diff
@@ -71,7 +71,7 @@
       expect(page).to have_css('[data-slider-object="1"] video', visible: false)
 
       within('.sul-embed-thumb-slider') do
-        expect(page).to have_css('.sul-embed-slider-thumb', count: 2)
+        expect(page).to have_css('.sul-embed-slider-thumb', count: 3)
```

Comment:
Just out of curiosity, why is this number dropping here?  What change is removing items from the thumb-slider?